docker-compose.yml

```diff
@@ -17,6 +17,34 @@ services:
             APP_SECRET:
     pattern-library:
         image: ${DOCKER_NAMESPACE}/pattern-library:${REVISION_PATTERN_LIBRARY}
+    content-store_fpm:
+        image: ${DOCKER_NAMESPACE}/content-store:${REVISION_CONTENT_STORE}
+        volumes:
+            - ./.docker/content-store/data:/app/data/
+        environment:
+            APP_SECRET:
+            DEFAULT_LOCALE:
+            SERVICE_NAME: scholarly-articles
+    content-store_web:
+        image: nginx:1.15.2-alpine
+        volumes:
+            - .docker/content-store/nginx.conf:/etc/nginx/conf.d/default.conf
+        healthcheck:
+            test: ["CMD", "nc", "-z", "localhost", "80"]
+            interval: 5s
+        depends_on:
+            - content-store_fpm
+    api-gateway:
+        image: nginx:1.15.2-alpine
```

Comment:
Do we need it fixed to `1.15.2` could we use `1.15-alpine` tag, staying up to date somewhat?

Reply:
I'd rather have fixed versions but automated updates (ie Dependabot).

Doesn't do Docker Compose (yet) unfortunately, but a local Dockerfile that doesn't do anything works.

Reply:
I matched what was already being used on `browser` and other applications, opinions?